tests/wdio.conf.js

```diff
@@ -173,8 +173,8 @@ const baseConfig = {
    * @param {Object} config wdio configuration object
    * @param {Array.<Object>} capabilities list of capabilities details
    */
-  onPrepare: async function (config) {
-    await utils.prepServices(config);
+  onPrepare: async function () {
+    await utils.prepServices({ suite: 'web' });
```

Comment:
Was this meant to be? I believe the suite check in `prepServices` means the config won't be uploaded.